CoffeeChat/Controller/EditingViewController.swift

```diff
@@ -280,8 +294,10 @@ class EditingViewController: UIViewController {
             NetworkManager.shared.updateUserGroups(groups: groups.map(\.name)).observe { result in
                 switch result {
                 case .value(let response):
+                    print(response)
                     if response.success {
                         print("Groups updated successfully")
+                        self.navigationController?.popViewController(animated: true)
```

Comment:
I think you want to access a `weak self` here, i.e. add `[weak self]` before `result in` on line 294 and then add a `?` after `self `on line 300.